test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -1077,5 +1077,24 @@ public void ParsingPropertyWithCoreTypeDefinitionAsPropertyTypeWorks()
             Assert.Equal(EdmTypeKind.TypeDefinition, property.Type.TypeKind());
             Assert.Equal("Org.OData.Core.V1.LocalDateTime", property.Type.FullName());
         }
+
+        [Theory]
+        [InlineData("4.0")]
+        [InlineData("4.01")]
+        public void ValidateEdmxVersions(string odataVersion)
+        {
+            string xml = "<?xml version=\"1.0\" encoding=\"utf-16\"?><edmx:Edmx Version=\"" + odataVersion + "\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\"><edmx:DataServices /></edmx:Edmx>";
+
+            var stringReader = new System.IO.StringReader(xml);
+            var xmlReader = System.Xml.XmlReader.Create(stringReader);
+
+            IEdmModel edmModel = null;
+            IEnumerable<EdmError> edmErrors;
+
+            // Read in the CSDL and verify the version
+            CsdlReader.TryParse(xmlReader, out edmModel, out edmErrors);
+            Assert.Equal(edmErrors.Count(), 0);
+            Assert.Equal(edmModel.GetEdmVersion(), odataVersion == "4.0" ? EdmConstants.EdmVersion4 : EdmConstants.EdmVersion401);

```

Comment:
>EdmConstants.EdmVersion4 [](start = 75, length = 24)

Can add the Enum into the [InlineData("4.0", EdmConstants.EdmVersion4)]

Reply:
I got an error from the test framework when I tried to add the static value to the InlineData.

---
In reply to: [290413640](https://github.com/OData/odata.net/pull/1476#discussion_r290413640) [](ancestors = 290413640)